packages/core/integration-tests/test/integration/globals/scope-hoisting.js

```diff
@@ -1 +1 @@
-output = require('./');
+global.output = require('./');
```

Comment:
Due to https://github.com/nodejs/node/issues/38918#issuecomment-1110258667 (only needed if the value being assigned is a function)